incubator/fluentd-elasticsearch/values.yaml

```diff
@@ -42,6 +42,12 @@ tolerations: {}
   #   operator: Exists
   #   effect: NoSchedule
 
+service: {}
```

Comment:
Does this service have to be generic? Why not call it `metricsService` and configure a single port instead of a range?

```yaml
metricsService:
  enabled: false
  port: 24231
  type: ClusterIP
```

Reply:
Yes, because the user could enable others plugin like [tcp input plugin](https://docs.fluentd.org/v0.12/articles/in_tcp) that will require more than one port

Reply:
So it's not just about exposing the Prometheus port?
Exposing the Prometheus port is just something this will enable and isn't what you are actually doing.
Should the PR be renamed to reflect that?

Reply:
ok, done